### ISSUE TITLE
Copy-DBADatabase - Improved output and runtime info

### ISSUE THREAD

**Issue by ChristopherSerre** [sqlcollaborative/dbatools#3821]
---
name: Feature request/idea :bomb: Copy-DBADatabase - Improved output and Runtime info
about: Suggest a new feature or enhancement

---

### Summary of new feature
Improved output for successful or failed migrated databases (including missing)

### Proposed technical details (if applicable)
Example of a migration via Copy-DBADatabase
Copy-DBADatabase -Source Xxx -Destination Yyy -BackupRestore -NetworkShare C:\Temp\Migration -Database 'test(2016)'
,'ThisDbExistsOnXxx','ThisDbIsMissingOnXxx','ThisDbIsNotCorectlySpelledOnXxx' 

Today’s Output
Type                     Name              Status     Notes
----                     ----              ------     -----
Database                 test(2016)        Skipped    Already exists 
Database (BackupRestore) ThisDbExistsOnXxx Successful       

More clear and better Output – This is my request
Type                     Name              		Status     Notes
----                     ----              		------     -----
Database                 test(2016)        		Skipped    Already exists 
Database (BackupRestore) ThisDbExistsOnXxx 		Successful    
Database (BackupRestore) ThisDbIsMissingOnXxx		Missing!    
Database (BackupRestore) ThisDbIsNotCorectlySpelledOnXxx 	Missing!        

Database Copy Summary (at the end as a summary):
Successfully Copied Databases = 1 of 4
Skipped Databases (Missing) = 2 of 4
Skipped Databases (Already Exists) = 1 of 4
Total Nr Of databases to be migrated = 4
Total Nr Of successfully migrated databases = 1 of 4
Runtime: 14min

Our Problem:
The problem we have today is that we sometimes get a long list of databases for migration (let’s say 28 databases) and the list is incorrect. Then it’s easy to miss that one database was not migrated because no info from the output.
The parameter “-Database” should check that all databases in the list exists and inform if something is missing. Maybe the script should not start to copy any of the databases before the list of databases is correct/ed?

### Latest version of dbatools as of writing
0.9.375

* Please provide the latest released version of dbatools module.
0.9.375

**Comment by wsmelton** [comment]
The output for all the copy commands was updated to output an object. If that is not sufficient please open a new issue with the details.
